validator/src/main/java/com/google/daq/mqtt/sequencer/sequences/ConfigSequences.java

```diff
@@ -32,7 +40,7 @@ public void system_last_update() {
     });
   }
 
-  @Test
+  @Test(timeout = 120000)
```

Comment:
why special-case the shorter time?

Reply:
As above, increased the global timeout so I can have a longer test, and then reinstated the shorter timeout on all the other tests